extra/bitcoin_interop.js

```diff
@@ -0,0 +1,66 @@
+var openpgp = require('openpgp');
```

Comment:
This whole bitcoin example is nice but i would keep it out of the patch. We will document it in a different way.

Let's evaluate the removal before finishing the integration of the patch